### ISSUE TITLE
`There should be no more than 1 consecutive blank lines.` should not trigger for string data

### ISSUE THREAD

**Issue by wkirschbaum** [rrrene/credo#1078]
### Environment

1.7.1-ref.main.3873d0cb+uncommittedchanges

Erlang/OTP 26 [erts-14.1] [source] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1] [jit:ns]

Elixir 1.15.6 (compiled with Erlang/OTP 26)

Linux/Arch

### What were you trying to do?

The following is valid code and often used when writing tests:

```elixir
  foo = "
  a


  b
  "
 ```

### Expected outcome

Credo should not emit a warning, as this is data not code. 

### Actual outcome

A code readability issue is shows in the report: 
`There should be no more than 1 consecutive blank lines.`

**Comment by rrrene** [comment]
That's right, that's a bug! 👍

**Comment by rrrene** [comment]
Thx for reporting it 🙂

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by wkirschbaum** [comment]
@rrrene thanks! It looks to be solved.
